src/strtab.rs

```diff
@@ -80,7 +80,13 @@ impl<'a> Strtab<'a> {
         let mut result = Self::from_slice_unparsed(bytes, offset, len, delim);
         let mut i = 0;
         while i < result.bytes.len() {
-            let string = get_str(i, result.bytes, result.delim)?;
+            let string = match get_str(i, result.bytes, result.delim) {
+                Ok(s) => s,
+                Err(_) => {
+                    // On error set to empty string
+                    ""
```

Comment:
while i think we should come up with a solution here, returning an empty string probably isn't ideal and wouldn't be useful for any kind of analysis if there is > 1 non-utf8 string in the binary